.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Be sure to do all of the following to help us incorporate your contribution
+quickly and easily:
+
+ - [ ] Make sure the PR title is formatted like:
+   `[CARBONDATA-<Jira issue #>] Description of pull request`
+ - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
+       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
+       number, if there is one.
+ - [ ] If this contribution is large, please file an Apache
+       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
```

Comment:
Additionally, If contribution is large, how about suggesting to file umbrella JIRA and raise smaller sub JIRAs? Many projects follow this. Always make smaller patches to get reviews easy?